CaloOnlineTools/HcalOnlineDb/src/HcalLutManager.cc

```diff
@@ -854,6 +854,84 @@ std::map<int, boost::shared_ptr<LutXml> > HcalLutManager::getLinearizationLutXml
   return _xml;
 }
 
+std::map<int, boost::shared_ptr<LutXml> > HcalLutManager::getHEFineGrainLUTs(std::string _tag, bool split_by_crate )
```

Comment:
Can the use of boost::shared_ptr be changed to std::shared_ptr or even std::unique_ptr throughout the code?

Reply:
The migration has been performed for `HcalLutManager.cc/.hh`. The other usage of `boost::shared_ptr` in `CaloOnlineTools/HcalOnlineDb/interface/LMap.h` is referenced in many places elsewhere in CMSSW and so should be resolved in a separate PR.